sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateHiveTableAsSelectCommand.scala

```diff
@@ -88,7 +108,9 @@ case class CreateHiveTableAsSelectCommand(
     } else {
       try {
         sparkSession.sessionState.executePlan(InsertIntoTable(
-          metastoreRelation, Map(), query, overwrite = true, ifNotExists = false)).toRdd
+        metastoreRelation, Map(), reorderOutputQuery, overwrite = true
+          , ifNotExists = false))
```

Comment:
nit: The comma should be in the line above (after `overwrite = true`). Actually I think we can put all the args to `InsertIntoTable` in the same line.